content/shpdealloc.tex

```diff
@@ -19,7 +19,7 @@
 \end{apiarguments}
 
 \apidescription{
-    SHPDEALLC returns a block of memory (allocated using \FUNC{SHPALLOC}) to the
+    SHPDEALLC returns a block of memory, allocated using \FUNC{SHPALLOC}, to the
```

Comment:
This is an essential clause, so the parentheses (old text) and commas (new text) should be dropped. That is, `SHPDEALLC` can only deallocate memory allocated using `SHPALLOC`, so the clause should not be offset.